requirements.txt

```diff
@@ -20,3 +20,4 @@ six==1.11.0
 git+https://github.com/armmbed/manifest-tool.git@v1.4.6
 mbed-cloud-sdk==2.0.1
 icetea>=1.0.2,<1.1
+idna==2.7 # Required by `icetea`
```

Comment:
I would really discourage us from doing things like this, especially if we don't actually depend on it in the tools contained within this repository. Is this requirement not already present in icetea?

Reply:
I didn't want to do this either, but as it is, Mbed OS is a lot more stricter with versions than icetea:
- https://github.com/ARMmbed/icetea/blob/master/requirements.txt
- https://github.com/ARMmbed/icetea/blob/master/dev_requirements.txt
- https://github.com/ARMmbed/icetea/blob/master/extra_requirements.txt

ARMmbed/icetea uses modules on the bleeding edge, wheras Mbed OS does not. The reason Mbed OS is squeaking along (imo) is that we cache our pip env in CI.

Reply:
I see. Hmm. Have we ever mentioned to the icetea to at least pin their major versions? (I realize you're just the messenger here, sorry for pestering).

Reply:
I hadn't, but have now: https://github.com/ARMmbed/icetea/issues/69

Priority was to get CI stable in Mbed OS first, as it seems that this is now affecting the release build...